zipkin/src/main/java/zipkin2/Span.java

```diff
@@ -25,14 +25,13 @@
 import java.util.Locale;
 import java.util.Map;
 import java.util.TreeMap;
-import java.util.logging.Logger;
```

Comment:
please revert all of this stuff in core.. I'll take another pass after. cheers and thanks for the help!